extensions/elytron-security-ldap/runtime/src/main/java/io/quarkus/elytron/security/ldap/config/DirContextConfig.java

```diff
@@ -26,12 +28,19 @@
     @ConfigItem
     public Optional<String> password;
 
+    /**
+     * how ldap redirects are handled
+     */
+    @ConfigItem(defaultValue = "ignore")
```

Comment:
Shouldn't it be `IGNORE`? Or is the config converter smart enough to uppercase that? /cc @dmlloyd @radcortez @geoand  ?

Reply:
Judging by the passing tests, I'd say that should be okay. +1 from me ;)

Reply:
I am pretty sure it will work as is

Reply:
Yeah, it was me recommending `ignore`. The values included for the documentation for enums are lowercased/kebab-cased. So whenever you define a default value for an enum, better use `default-value` rather than `DEFAULT_VALUE`. This way, it's consistent with the list of available values.

Reply:
Thanks for the explanation.